docs/docs/20-usage/45-cron.md

```diff
@@ -0,0 +1,29 @@
+# Cron
+
+To enable cron jobs you need at least have push access to the repo.
```

Comment:
```suggestion
To configure cron jobs you need at least push access to the repo.
```

Reply:
Currently the UI only allows repo admins to access the settings area if I remember correctly. I saw that recently that this access is not matching the permissions defined by the router.